contrib/google.golang.org/grpc/client.go

```diff
@@ -182,7 +182,9 @@ func doClientRequest(
 	if methodKind != "" {
 		span.SetTag(tagMethodKind, methodKind)
 	}
-
+	if host, _, err := net.SplitHostPort(cc.Target()); err == nil {
+		span.SetTag(ext.PeerHostname, host)
+	}
```

Comment:
Any way that `cc` could be nil? If so, this could panic.

Reply:
I don't think `cc` should be `nil` in any case where this code is being executed, but added a check just in case.